components/styleguide/image-styles/image.css

```diff
@@ -1,33 +1,29 @@
 /*
-Image Field Type
+Image
```

Comment:
Should this be somewhere else? I think this group should be probably called "Image styles"

Reply:
This has now been renamed to image-styles.css and moved back to the components directory.